<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TreeSearchDlg</class>
 <widget class="QDialog" name="TreeSearchDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>1200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Search...</string>
  </property>
  <widget class="QProgressBar" name="m_progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>471</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>24</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QTableWidget" name="m_SearchResults">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>471</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_checkCurrentTree">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>142</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Only in current tree</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_checkSearchInAttach">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>200</y>
     <width>159</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Search in attachments</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_checkSearchInANames">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>180</y>
     <width>193</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Search in attachment names</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_checkSearchInDescriptor">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>150</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Search in node descriptors</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_checkSearchInNode">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>152</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>21</height>
    </size>
   </property>
   <property name="text">
    <string>Search in node names</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_checkWholeWords">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>128</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Whole words only</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_checkRegExp">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>164</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Use regular expressions</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_checkCaseSensitive">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>110</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>21</height>
    </size>
   </property>
   <property name="text">
    <string>Case sensitive</string>
   </property>
  </widget>
  <widget class="QLabel" name="LblSearchOptions">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16</height>
    </size>
   </property>
   <property name="text">
    <string>Search options:</string>
   </property>
  </widget>
  <widget class="QLabel" name="LblSearchResults">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>24</height>
    </size>
   </property>
   <property name="text">
    <string>Search results:</string>
   </property>
  </widget>
  <widget class="QLabel" name="LblFake">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>56</width>
     <height>13</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>13</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>13</height>
    </size>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="LblFake2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>56</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="SearchLabel">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Search:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_eSearchString">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>421</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>421</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="m_bHide">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>460</y>
     <width>118</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>118</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>27</height>
    </size>
   </property>
   <property name="text">
    <string>Hide</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_bClearResult">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>460</y>
     <width>118</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>118</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>27</height>
    </size>
   </property>
   <property name="text">
    <string>Clear results</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_bSearch">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>460</y>
     <width>118</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>118</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>27</height>
    </size>
   </property>
   <property name="text">
    <string>Begin search</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_lblPatternSyntax">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pattern syntax</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbPatternSyntax">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>411</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="m_dateTimeEdit_from">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="m_dateTimeEdit_to">
   <property name="geometry">
    <rect>
     <x>403</x>
     <y>260</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_checkBoxFrom">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>85</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Created after:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_checkBoxTo">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>230</y>
     <width>85</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Created before:</string>
   </property>
  </widget>
  <zorder>SearchLabel</zorder>
  <zorder>m_eSearchString</zorder>
  <zorder>m_bHide</zorder>
  <zorder>m_bClearResult</zorder>
  <zorder>m_bSearch</zorder>
  <zorder>m_SearchResults</zorder>
  <zorder>m_checkCurrentTree</zorder>
  <zorder>m_checkSearchInAttach</zorder>
  <zorder>m_checkSearchInANames</zorder>
  <zorder>m_checkSearchInDescriptor</zorder>
  <zorder>m_checkSearchInNode</zorder>
  <zorder>m_checkWholeWords</zorder>
  <zorder>m_checkRegExp</zorder>
  <zorder>m_checkCaseSensitive</zorder>
  <zorder>LblSearchOptions</zorder>
  <zorder>LblSearchResults</zorder>
  <zorder>LblFake</zorder>
  <zorder>m_progressBar</zorder>
  <zorder>LblFake2</zorder>
  <zorder>m_lblPatternSyntax</zorder>
  <zorder>m_cmbPatternSyntax</zorder>
  <zorder>m_dateTimeEdit_from</zorder>
  <zorder>m_dateTimeEdit_to</zorder>
  <zorder>m_checkBoxFrom</zorder>
  <zorder>m_checkBoxTo</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
